releases/v1.7.4.toml

```diff
@@ -0,0 +1,37 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+# project_name is used to refer to the project in the notes
+project_name = "containerd"
+
+# github_repo is the github project, only github is currently supported
+github_repo = "containerd/containerd"
+
+# match_deps is a pattern to determine which dependencies should be included
+# as part of this release. The changelog will also include changes for these
+# dependencies based on the change in the dependency's version.
+match_deps = "^github.com/(containerd/[a-zA-Z0-9-]+)$"
+
+# previous release of this project for determining changes
+previous = "v1.7.3"
+
+# pre_release is whether to include a disclaimer about being a pre-release
+pre_release = false
+
+# preface is the description of the release which precedes the author list
+# and changelog. This description could include highlights as well as any
+# description of changes. Use markdown formatting.
+preface = """\
+The fourth patch release for containerd 1.7 contains remote differ plugin support,
+a new block file based snapshotter, and various fixes and updates.
+
+### Notable Updates
+
+* **Add blockfile snapshotter** ([#8986](https://github.com/containerd/containerd/pull/8986))
+* **Add remote/proxy differ** ([#8985](https://github.com/containerd/containerd/pull/8985))
+* **Update runc binary to v1.1.9** ([#8965](https://github.com/containerd/containerd/pull/8965))
+* **Cri: Don't use rel path for image volumes** ([#8926](https://github.com/containerd/containerd/pull/8926))
+* **Allow attaching to any combination of stdin/out/err** ([#8910](https://github.com/containerd/containerd/pull/8910))
+* **Fix ro mount option being passed** ([#8887](https://github.com/containerd/containerd/pull/8887))
```

Comment:
Would you mind to add the item about for [*: fix leaked shim caused by high IO pressure](https://github.com/containerd/containerd/pull/9003)? Thanks

```
* ***: fix leaked shim caused by high IO pressure** ([#9003](https://github.com/containerd/containerd/pull/9003))
```

Reply:
Of course yea, adding

Reply:
Forgot to get this https://github.com/containerd/containerd/pull/9005 😂 Hope we can consider this one. It can work with #9003

Reply:
I'll draft this for now, it's in no rush. Lets get in your changes

Reply:
Well, I'll draft in an hour or so. Seems you can't do it on mobile (or I can't find it 😆)

Reply:
Yeah. Both are merged!

Reply:
Perfect, let me update this